packages/monaco-graphql/src/monaco-editor.ts

```diff
@@ -19,4 +19,14 @@
 import 'monaco-editor/esm/vs/basic-languages/graphql/graphql.contribution.js';
 import 'monaco-editor/esm/vs/language/json/monaco.contribution.js';
 
-export * from 'monaco-editor/esm/vs/editor/edcore.main.js';
```

Comment:
`edcore.main.js` imports `editor.all.js`, which includes many features we don't even use yet!

instead, we implement only the minimum features we need for our mode to work, and users can opt in to the rest